docs/source/serving/load_balancing.rst

```diff
@@ -0,0 +1,78 @@
+.. _load_balancing:
+
+Load Balancing
+===============
+
+In scenarios where multiple GPUs or nodes are necessary to handle the model serving workload, it is advisable to initiate several vLLM server instances and distribute incoming requests amongst them using an HTTP load balancer.
```

Comment:
Can you be a bit more clear about the use case here? Just say something like "You can run multiple vLLM instances each on a separate set of GPUs and use a load balancer to distribute the requests."